parser-typechecker/src/Unison/Codebase/SqliteCodebase/Operations.hs

```diff
@@ -554,32 +553,22 @@ referentsByPrefix doGetDeclType (SH.ShortHash prefix (fmap Cv.shortHashSuffix1to
   pure . Set.fromList $ termReferents <> declReferents
 
 -- | Get the set of branches whose hash matches the given prefix.
-causalHashesByPrefix :: ShortCausalHash -> Transaction (Set Branch.CausalHash)
+causalHashesByPrefix :: ShortCausalHash -> Transaction (Set CausalHash)
 causalHashesByPrefix sh = do
   -- given that a Branch is shallow, it's really `CausalHash` that you'd
   -- refer to to specify a full namespace w/ history.
   -- but do we want to be able to refer to a namespace without its history?
-  cs <- Ops.causalHashesByPrefix (Cv.sch1to2 sh)
-  pure $ Set.map (Causal.CausalHash . unCausalHash) cs
-
--- returns `Nothing` to not implemented, fallback to in-memory
---    also `Nothing` if no LCA
--- The result is undefined if the two hashes are not in the codebase.
--- Use `Codebase.lca` which wraps this in a nice API.
-sqlLca :: Branch.CausalHash -> Branch.CausalHash -> Transaction (Maybe Branch.CausalHash)
```

Comment:
Did you mean to delete this? Presumably it's unused, but seems unrelated to just removing the extra set of hashes.

Reply:
Yeah, I did mean to delete it - after unifying the two `CausalHash` types it just became `sqlLca = lca` (from the other `Operations` module)